packages/website/package.json

```diff
@@ -35,32 +35,21 @@
     "@typescript-eslint/eslint-plugin": "^2.33.0",
     "@typescript-eslint/parser": "^2.34.0",
     "@vue/cli": "^4.3.1",
-    "@vue/cli-plugin-babel": "^4.3.1",
     "@vue/cli-plugin-eslint": "^4.3.1",
     "@vue/cli-plugin-typescript": "^4.3.1",
     "@vue/cli-service": "^4.3.1",
-    "@vue/component-compiler-utils": "^3.1.2",
     "@vue/eslint-config-airbnb": "^5.0.2",
     "@vue/eslint-config-prettier": "^6.0.0",
     "@vue/eslint-config-typescript": "^5.0.1",
-    "autoprefixer": "^9.8.0",
-    "babel-plugin-transform-vue-jsx": "^3.7.0",
-    "core-js": "^3.6.5",
     "eslint": "^7.0.0",
-    "eslint-config-airbnb-base": "^14.1.0",
     "eslint-import-resolver-typescript": "^2.0.0",
-    "eslint-import-resolver-webpack": "^0.12.1",
```

Comment:
This I'm not 100% sure on, I think it was needed to support JS components still but I think we removed all non-trivial ones.

Reply:
It was probably there to support `@` path aliasing

Reply:
Indeed it was.

Reply:
(TypeScript handled @ paths for TypeScript components, I believe, but JS components still needed it)